flow/Makefile

```diff
@@ -1050,7 +1050,7 @@ yosys:
 # Drop into a bash shell with all environment variables, useful for debugging
 .PHONY: bash
 bash:
-	bash
+	export MAKEFILE_ENV=1; bash
```

Comment:
Why not just do:

```
bash:
  bash --init_file $(FLOW_HOME)/../env.sh
```

And avoid touching bashrc?

Reply:
Neat trick, thanks. I am using the MAKEFILE_ENV variable still to identify that `make bash` is called, is that okay?

Reply:
Well #1497 is only asking to fix the prompt for `make bash`. Maybe it's just easier to do the following?

```
bash:
  bash --init_file <(echo '...')
```

This uses a pipe redirect so there is no external file needed. the `...` is the `PS1=...` string with all the correct character escapes.

Reply:
Fixed, should be cleaner now.